js/plates-model/fields-collision.js

```diff
@@ -41,13 +41,18 @@ export default function fieldsCollision (bottomField, topField) {
 
   if (bottomField.isOcean) {
     subduction(bottomField, topField)
-    if (bottomField.isContinentBuffer && !topField.isContinent) {
+    if (bottomField.isContinentBuffer && !topField.isContinent && !topField.isContinentBuffer) {
       // Special case when the continent is "trying" to subduct under the ocean. Apply drag force to stop both plates.
+      // There's one exception - when both fields are continent buffers, it means continents are about to collide soon.
+      // Don't apply forces in this case, so the orogeny can actually happen.
       applyDragForces(bottomField, topField)
     }
   } else if (bottomField.isContinent) {
     if (topField.isContinent) {
       orogeny(bottomField, topField)
+    } else if (topField.isContinentBuffer) {
+      // Continents are next to each other. They will collide soon. Remove oceanic field to let that happen.
+      topField.alive = false
```

Comment:
To clarify: does removing the top field here actually change the behavior of the collision? Or just make the collision look better.

Reply:
It does. We remove ocean crust that otherwise would collide with the continent and won't allow orogeny to happen.